backend/clubs/views.py

```diff
@@ -2848,6 +2850,18 @@ def destroy(self, request, *args, **kwargs):
                 f"If you would like to do this, email {settings.FROM_EMAIL}."
             )
 
+        if (
+            Ticket.objects.filter(event=event)
+            .filter(Q(owner__isnull=False) | Q(holder__isnull=False))
+            .exists()
+        ):
+            raise DRFValidationError(
```

Comment:
is a `DRFValidationError` appropriate?

Reply:
For consistency, I think this is fine. According to the [docs](https://www.django-rest-framework.org/api-guide/exceptions/#validationerror), the API will return status code 400, which is appropriate here